src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/addBook';
+const REMOVE_BOOK = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+// Add Reducers
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return { ...state };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        items: state.items.filter((item) => item.id !== action.id),
+      };
```

Comment:
- [ ] Kindly make sure to filter from the correct array because at this moment you are filtering from an unexistent array `state.items` instead of `state.books`. Kindly fix that to avoid future errors once you connect your redux to react components.